python/lvmdrp/core/sky.py

```diff
@@ -740,9 +767,11 @@ def get_telescope_shadowheight(header, telescope):
     if telescope not in {"SKYE", "SKYW", "SCI", "SPEC"}:
         raise ValueError(f"invalid value for 'telescope' parameter: '{telescope}', valid values are 'SKYE', 'SKYW', 'SCI', or 'SPEC'")
 
-    header["JD"] = header["MJD"] + 2400000.5
+    #header["JD"] = header["MJD"] + 2400000.5
     ra, dec = header[f"TE{telescope}RA"], header[f"TE{telescope}DE"]
-    jd = header["JD"]
+    #jd = header["JD"]
+    time = Time(header["OBSTIME"],format='isot', scale='utc')
```

Comment:
remove the old code if you no longer need it

Reply:
done